tea.go

```diff
@@ -147,24 +146,17 @@ type Program struct {
 	renderer      renderer
 
 	// where to read inputs from, this will usually be os.Stdin.
-	input        io.Reader
+	input io.Reader
+	// tty is null if input is not a TTY.
+	tty          *os.File
+	ttyState     *term.State
```

Comment:
this could prob be called `previousTtyState` or something like that, since it is the state the term had before `MakeRaw` 🤔